docs/getting-started/tutorial.md

```diff
@@ -25,7 +25,7 @@ To complete this tutorial, you will need:
 If you don't have one or more of these items, the [Block Development Environment](https://developer.wordpress.org/block-editor/getting-started/devenv/) documentation will help you get started. Come back here once you are all set up.
 
 <div class="callout callout-info">
-	This tutorial uses <a href="https://developer.wordpress.org/block-editor/getting-started/devenv/get-started-with-wp-env/"><code>wp-env</code></a> to create a local WordPress development environment. However, feel free to use alternate local development tools if you already have one that you prefer.
+	This tutorial uses <a href="https://developer.wordpress.org/block-editor/getting-started/devenv/get-started-with-wp-env/"><code>wp-env</code></a> to create a local WordPress development environment. However, feel free to use any development environment that meets the abovementioned prerequisites.
```

Comment:
needs a space between above and mentioned :)

Reply:
Thanks for the feedback! :)
I'm happy to make the change but are we sure about this?
I just double-checked several online sources, they all say that it is "abovementioned" in American English and "above-mentioned", with a hyphen, in British English. :)

Reply:
ahh, thank you for pointing this out. Even as a native English speaker, I was not aware of this; you are correct. (https://www.merriam-webster.com/dictionary/abovementioned 

WordPress uses American English spelling, per https://make.wordpress.org/docs/style-guide/language-grammar/word-choice/

Reply:
Thanks for the link to the Style Guide. The "Word list and usage dictionary" mentioned there will surely come in handy. :)

Reply:
TIL too!

Reply:
I'm new here wanna learn more